boardgame-ui/src/components/board_games/AddBoardGameModal.vue

```diff
@@ -132,11 +140,30 @@ export default {
       let bggId = Number(boardGame.bgg_id || boardGame.id);
       let inLibrary = this.bggIdsLibraryGames.includes(bggId);
       this.$emit('add', {bggId, inLibrary});
+    },
+
+    countWished(boardGame) {
```

Comment:
Good for now as we don't have much board games and wishes, but might need optimization at some point. It makes the template rendering O(b * w) where b is the number of board games to display and w the number of wishes.

Reply:
I am reworking to call this function only once by board game. Note that the number of games can be pretty large when performing a search on BGG, so if you have suggestions for further improving it, they are welcome.

Reply:
You could count once and for all the wished games with something like this (not tested and can probably be made more compact): 


```javascript
[...]
computed: {
  [...],
  wishCounts() {
    if(!this.wishedBoardGames) {
      return {};
    }
    let counts = {};
    this.wishBoardGames.forEach(boardgame => {
      if (!counts[boardgame.bgg_id]) {
        counts[boardgame.bgg_id] = 1;
      } 
      else {
        counts[boardgame.bgg_id] += 1;
      }
    });
    return counts; 
  }
},
[...]
methods: {
  countWished(boardGame) {
    let bggId = Number(boardGame.bgg_id || boardGame.id);
    return this.wishCounts[bggId] ? this.wishCounts[bggId] : 0;
  }
}
```

Reply:
Indeed, way better, thanks! :-)